src/test/java/testPackage/appium/DragAndDropTest.java

```diff
@@ -0,0 +1,38 @@
+package testPackage.appium;
+
+import com.shaft.driver.SHAFT;
+import io.appium.java_client.AppiumBy;
+import io.appium.java_client.remote.AutomationName;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Platform;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class DragAndDropTest {
+    private SHAFT.GUI.WebDriver driver;
+    private final AppiumBy dragBtn = new AppiumBy.ByAndroidUIAutomator("UiSelector().text(\"Drag\")");
+    private final By toBeDragged =  By.xpath("//android.view.ViewGroup[@content-desc=\"drag-l1\"]/android.widget.ImageView");
+    private final By toBeDropped =By.xpath("//android.view.ViewGroup[@content-desc=\"drop-l1\"]/android.view.ViewGroup");
+    @BeforeMethod
+    public void setupUp(){
+        System.setProperty("mobile_autoGrantPermissions", "true");
```

Comment:
Please note that any properties you set in your test must be unset after it. Else it will impact the rest of the execution pipeline.
I suggest using before/after configuration methods.

Also kindly make sure that your new test class is excluded from the desktop/web testing jobs, and included in the android native testing jobs.

You'll find the full pipeline configuration under the .github folder

Reply:
Okay I'll work on it today In-sha-Allah

Thanks.

Reply:
@MustafaAgamy @MahmoudElSharkawy are we ready to merge?

Reply:
I wanted to ask about the Pipeline part
I already mentioned it on slack 😅
If it's not clear yet maybe we can Remove the dragAndDrop test for now

Reply:
Kindly comment the class for now